app/dashboard/utils/feed/soc.py

```diff
@@ -319,9 +318,6 @@ def soc_feed(soc):
             {
                 "href": FRONTEND_SOC_JOB_URL, "label": u"SoC boot reports"
             },
-            {
-                "href": BOOT_JOB_REPORTS_URL, "label": u"All boot reports"
-            },
```

Comment:
Maybe this one should be removed separately - it's the feed, not a boot view.

Reply:
Removed.